### PR TITLE
Issue-23: Improve the sentences, make them short and concise.

### PR DESCRIPTION
**Pull request by dostiharise**
- Rephrased many of the sentences.

### CODE REVIEW

File: content/b-overview/1-consulting.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ weight: 1
 
 ### Software Consulting
 
-We are Quality focused Software Consulting Experts who can Design, Build, and Scale your Internet Applications, for the modern era.
+Quality focused Software Consulting Experts.
+
+We can Design, Build, and Scale your Internet Applications, for the modern era.
```
Comment (milindmantri) [inline content/b-overview/1-consulting.md:11]: IMO, boldening the words `design`, `build` and `scale` might look better.

File: content/d-products/1-cocoon.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ weight: 1
 
 ### Alvyl Cocoon
 
-Alvyl Cocoon is a product offering for Internet of Things (IoT), built by Alvyl Consulting.
+Alvyl Cocoon is an Internet of Things (IoT) product.
 
-Cocoon provides cloud hosted services for acquisition of real-time data from Industrial Machines and managing IoT Edge Devices, at Scale.
+Cocoon provides Edge and Cloud hosted services for acquisition of real-time data from Industrial Machines and managing IoT Edge Devices, at Scale.
```
Comment (milindmantri) [inline content/d-products/1-cocoon.md:11]: `Scale` -> `scale`

File: content/d-products/2-loom.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ weight: 2
...
 ### Alvyl Loom
 
-Alvyl Loom is a platform offering, for performing Data Analysis & Analytics, from Alvyl Consulting.
+Alvyl Loom is a platform for Data Analysis & Analytics.
 
 Loom offers capabilities that meet the Storage, Analysis & Retrieval needs of modern high volume Data environments, using Open Source technology stacks, that can Scale.
 
-Loom can integrate and ingest data from Alvyl Cocoon for analysis IoT data.
\ No newline at end of file
+It can integrate and ingest data from Alvyl Cocoon for analysis IoT data.
```
Comment (milindmantri) [inline content/d-products/2-loom.md:13]: `... for analysing IoT data.`

File: content/e-services/1-build.md [hunk 4]
```diff
@@ -6,12 +6,15 @@ weight: 1
 
 ### Build
 
-Core at heart we are builders, who practice craft of making software. We use computing technologies to solve problems for humans.
+Core at heart we are builders.
```
Comment (milindmantri) [inline content/e-services/1-build.md:9]: Doesn't `core` imply `heart`?

File: content/e-services/5-scale.md [hunk 5]
```diff
@@ -6,12 +6,13 @@ weight: 5
...
 We adopt many awesome Open Source technologies and Public Cloud Providers to make this possible.
 
 Some of the Services & Platforms we used to scale are:
 
-Nginx, HaProxy, 
-Cassandra, 
-Docker, Kubernetes, Helm, 
-Amazon Web Services (AWS), Microsoft Azure, Google Cloud Platform
\ No newline at end of file
+- Nginx, HaProxy
```
Comment (milindmantri) [inline content/e-services/5-scale.md:15]: `HaProxy` -> 'HAProxy`

**Review by milindmantri — COMMENTED** [review]
Have suggested minor changes.
Please take a look

File: content/d-products/1-cocoon.md (hunk 2 above)
Comment (dostiharise) [reply content/d-products/1-cocoon.md:11]: I think if I try and bold them, I don't have to rely on capital `Scale`. That way I can stay coherent with grammer, and highlight.

Thanks for the comments, and hence indirectly the idea. :-)

File: content/e-services/1-build.md (hunk 4 above)
Comment (dostiharise) [reply content/e-services/1-build.md:9]: :-) Not sure. Let me lookup what Internet says.

**Pull request merged**